docs/usage/configuration-options.md

```diff
@@ -410,7 +410,7 @@ This is an advance field and it's recommend you seek a config review before appl
 
 ## bumpVersion
 
-Currently this setting supports `helmv3`, `npm`, `nuget`, `maven`, `pep621`, `poetry` and `sbt` only, so raise a feature request if you have a use for it with other package managers.
+Currently, this setting supports `helmv3`, `npm`, `nuget`, `maven`, `ocb`, `pep621`, `poetry` and `sbt` only, so raise a feature request if you have a use for it with other package managers.
```

Comment:
We shold convert this to a list. @HonkingGoose WDYT?

Reply:
Let's put this in a list. 🙂